### ISSUE TITLE
Velocity Indicator 

### ISSUE THREAD

**Issue by mdsmestad** [endless-sky/endless-sky#1248]
This would be a nice feature as part of the hub, to gauge how fast you've gotten your ship going.

**Comment by Bladewood** [comment]
...I want this.

**Comment by Alkallid** [comment]
Didn't the guy with the radar picture already ask for this? Name sounded like Wambat's?

**Comment by Amazinite** [comment]
@alkallid you mean wargwargwarg?

**Comment by Alkallid** [comment]
Yup.

**Comment by wambat1** [comment]
this would be useful when you are trying to keep your fleet grouped so you can limit your speed to that of your slowest ships.

**Comment by mdsmestad** [comment]
@wambat1 And it would just be fun to see how fast your going w/ your new engines

**Comment by XHawk87** [comment]
...or when you're thrown off at great speed by an explosion or barrage of particle cannons

**Comment by Alkallid** [comment]
Well, you can tell your max speed in an outfitter, and it's easy to estimate from being stationary to full, and where would it go in the HUD?

**Comment by jafdy** [comment]
Could put it in the info wheel.

**Comment by Alkallid** [comment]
Not helpful, as speed is very dynamic and pausing to check speed doesn't really help.

**Comment by jafdy** [comment]
The ring you can put round your ship that shows your hull and shields.

**Comment by IFailAtGaming** [comment]
I think it would look cooler around the mini map radar thingamajig (that's an actual word apparently)

**Comment by LocalGod79** [comment]
i agree with @Alkallid - what *real* purpose does this serve? how have we managed without this feature for so long?

in the end, i believe the answer to the first question is: clutter.

**Comment by Amazinite** [comment]
But I need to know how fast [this](www.steamcommunity.com/sharedfiles/filedetails/?id=699361379) is. 

All jokes aside, this doesn't really feel like it would actually add anything to the game other than eye candy.

**Comment by Alkallid** [comment]
"Move, asteroid, get out the way" speed.

**Comment by LocalGod79** [comment]
@Amazinite everything is whited out anyway - you wouldn't be able to see the display.

and as @Alkallid pointed out, even when you're moving, the number would be changing so frequently that it wouldn't really be useful in any circumstance except to reveal max afterburner speed (assuming a reasonable installation), and to advertise that weapons recoil can make you go faster than even that...

**Comment by wambat1** [comment]
i thought i gave a practical application above, but it takes just a little imagination to come up with more: now that you will no longer be able to stupefy other ships by requesting fuel, some players will want to specialize in pirating by first luring ships away from system center, but then kiting and out-ranging them. they will become familiar with certain high value targets and the top speed of that target, and will want to approximately match that top speed while kiting.
a speed indicator is not essential for this scenario, but is definitely an enhancement. we already have examples of things that are not of much practical use to the player, but which a player would not like to be without. after quite a lot of hours of game play i am convinced i could l play the game every time, all the way through, without knowing my net worth. as far as i can tell, it seems to be there to make the player feel better about himself.
this speed indicator might serve largely the same purpose, but it could be a very valuable purpose for the significant number of players who seem interested in pushing the speed envelop. having this feature might help significantly boost the number of screen shots for people showing off their speedy ships, which should help in publicizing the game.

presumably, the game knows how fast your ship is going at any time, so the information to be displayed should beaccessible. much of the time the player will be going at a constant speed

(max speed, coasting) and limits can be set to how often the displayed number is revised to that is in not an indecipherable blur.
    On Monday, June 20, 2016 11:08 AM, LocalGod79 <notifications@github.com> wrote:
 

 @Amazinite everything is whited out anyway - you wouldn't be able to see the display.and as @Alkallid pointed out, even when you're moving, the number would be changing so frequently that it wouldn't really be useful in any circumstance except to reveal max afterburner speed (assuming a reasonable installation), and to advertise that weapons recoil can make you go faster than even that...—

**Comment by mdsmestad** [comment]
@LocalGod79 I agree with @wambat1 for the same reasons stated above. It's just really nice to see it. I also think theres the vanity to it. Players like to see there new ship get a speed bump and this gives you something more material to see.

**Comment by Alkallid** [comment]
Then this might get oushed into 1.0.

**Comment by VonZorch** [comment]
I was going to ask for something similar, a vector arrow in the flagship display (upper right corner).  Originating in the center it would point in the direction your ship is moving and it's length would proportional to speed with ships maximum speed  at the hull ring.

**Comment by tehhowch** [comment]
Velocity magnitude could be something provided in the tactical scan display. Would need both sprite work and a code change.

**Comment by warp-core** [comment]
> Velocity magnitude could be something provided in the tactical scan display. Would need both sprite work and a code change.

Given that the tactical scan display shows details about your target, this doesn't seem like a great idea. Relative velocity of your target, though...

**Comment by Zitchas** [comment]
For added complexity - how about both? I'd love to have a straight up velocity indicator; but having one that shows relative to one's target would also be rather interesting.

**Comment by TheGiraffe3** [comment]
It's worth noting that [Endless Sky Delta](https://github.com/endless-sky/Endless-Sky-Delta) has this feature, among others.
Theoretically that could be taken out and used here.
